apps/docs/src/app/core/component-docs/radio/examples/radio-example.component.html

```diff
@@ -1,125 +1,266 @@
         </div>
         <div fd-form-item [isInline]="true">
-            <label fd-form-label for="radio-9" [radio]="true">
-                <input fd-form-control type="radio" id="radio-9" name="radio-name-3" value="">
-                Option Three
-            </label>
+            <fd-radio-button [value]="'val3'" id="radio-9" name="radio-name-3" [(ngModel)]="optionThreeVariable">
+                Option One
+            </fd-radio-button>
         </div>
     </fd-form-group>
 </fieldset>
 
-<fieldset fd-fieldset>
+<fieldset fd-form-set>
     <legend fd-form-legend>Radio button States</legend>
     <fd-form-group>
-        <div fd-form-item>
-            <label fd-form-label for="radio-10" [radio]="true">
-                <input fd-form-control type="radio" id="radio-10" name="radio-name-4" [state]="'valid'">
+        <div fd-form-item [isCheck]="true">
+            <fd-radio-button
+                name="radio-name-4"
+                id="radio-12"
+                [value]="'val0'"
+                [state]="'default'"
+                [(ngModel)]="optionFourVariable"
+            >
+                Default Option
+            </fd-radio-button>
+        </div>
+        <div fd-form-item [isCheck]="true">
+            <fd-radio-button
+                id="radio-10"
+                name="radio-name-4"
+                [state]="'valid'"
+                [value]="'val1'"
+                [(ngModel)]="optionFourVariable"
+            >
                 Valid Option
-            </label>
+            </fd-radio-button>
         </div>
-        <div fd-form-item>
-            <label fd-form-label for="radio-11" [radio]="true">
-                <input fd-form-control type="radio" id="radio-11" name="radio-name-4" [state]="'warning'">
+        <div fd-form-item [isCheck]="true">
+            <fd-radio-button
+                name="radio-name-4"
+                id="radio-11"
+                [value]="'val2'"
+                [state]="'warning'"
+                [(ngModel)]="optionFourVariable"
+            >
                 Warning Option
-            </label>
+            </fd-radio-button>
         </div>
-        <div fd-form-item>
-            <label fd-form-label for="radio-12" [radio]="true">
-                <input fd-form-control type="radio" id="radio-12" name="radio-name-4" [state]="'invalid'">
+        <div fd-form-item [isCheck]="true">
+            <fd-radio-button
+                name="radio-name-4"
+                id="radio-12"
+                [value]="'val3'"
+                [state]="'invalid'"
+                [(ngModel)]="optionFourVariable"
+            >
                 Invalid Option
-            </label>
+            </fd-radio-button>
         </div>
-        <div fd-form-item>
-            <label fd-form-label for="information-input" [radio]="true">
-                <input fd-form-control type="radio" id="information-input" name="radio-name-4" [state]="'information'">
+        <div fd-form-item [isCheck]="true">
```

Comment:
Should we still use `fd-form-item [isCheck]="true"` for custom radio controls?

Reply:
already removed